tests/io/lobster/test_inputs.py

```diff
@@ -1841,14 +1839,14 @@ def test_dict_functionality(self):
         assert len_after == len_before - 1
 
         # Test case sensitivity of |= operator
-        self.Lobsterin |= {"skipCOHP": True}  # Camel case
+        self.Lobsterin["skipCOHP"] = True  # Camel case
```

Comment:
This was intended for testing the case insensitivity of `|=` operator, should not have been changed :)

Reply:
oops, sorry about the extra work. can you revert in your next PR?

Reply:
Sure :) No problem. And thanks for reviewing and the `PymatgenTest` replacement